flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/MapJoinLeftRunner.scala

```diff
@@ -31,7 +32,19 @@ class MapJoinLeftRunner[IN1, IN2, OUT](
   override def flatMap(multiInput: IN1, out: Collector[OUT]): Unit = {
     broadcastSet match {
       case Some(singleInput) => function.join(multiInput, singleInput, out)
-      case None =>
+      case None => {
```

Comment:
We cant pass null value to join function, but adding of `outerJoin` parameter is useful. It will fix cross join testcase.
Thanks!!!